public/stats-faq.md

```diff
@@ -0,0 +1,15 @@
+# STATS
+
+- __How are commits counted?__
+
+  - Commits are counted only after your PR is merged.
+  - Commits are counted if they are merged in the branch opted by mentor.Mentor is given the option to select the branch and can change it any time during the event.
+  - Commits are not linked to your github account if your email configuration is not set properly. See [this](https://github.com/kossiitkgp/kwoc-bugs/issues/22#issuecomment-747068412) for details.
+
+- __What to do,if my stats are not being updated?__
+
+  Wait for 24 hours,then contact us via admin@kossiitkgp.org.
+  
+ ###  PS - The main aim of stats was to provide encouragement but not create a competitive atmosphere.
+ 
+>  You won't get sued for anticompetitive behavior. - Linus Torvalds.
```

Comment:
Rather than making a new file, we should put these questions in the `faq.md` but under a different heading.

Reply:
done